### DISCUSSION TITLE
Agenda for 05/10/22

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/media#5] [General]
- Quick Retro: and milestone reflection https://github.com/omigroup/media/milestones?with_issues=no
- Discussion Review: What are the goals of this group? https://github.com/omigroup/media/discussions/7
- Content Sprint #3 (primary effort)
- Touchbase: Can we get help with #6? 
- Touchbase: https://github.com/omigroup/media/discussions/17

**Comment by pcsmgnt** [comment]
Meeting Minutes: 10/5/22 

**In attendance:**
Jin, Anthony

**Apologies:**
Jesse, Evo

_A - Outstanding Permissions_
-  Create official links channel for media conversations in Discord
- Embed links using Mee6 bot to complete some actions *(below)* . The feature *may* not be avail. on free version
- Create Discord channel for internal comms and explainer videos

_B - Pending Actions_
- Post socials for easy access and reference. Due: 11/05 (@littlemissbold)
- Post OMI One year post (@antpb#0319 - Due:  10/05 | Request for approval 11/05)
- How to become and contributor" explainer video (Anthony @antpb#0319  - Due: TBC)
- Submit content to Airtable incl. Metaverse article (@ZRO#1080, @LMB | LittleMissBold#1872 - Due 11/05)

_C - Update outcomes of Champions w/ 3 Chairs on Friday 6th May)*** @evo#6444 @Robert#0293 @mrmetaverse#0001  Due: 11/05_
- Editorial strategy
- Volunteers
